Classes/PackagesTYPO3ExtensionsGenerator.php

```diff
@@ -104,6 +104,9 @@ protected function getPackageArray(SimpleXMLElement $extension, SimpleXMLElement
 				'url' => 'http://typo3.org/extensions/repository/download/' . $extension['extensionkey'] . '/' . $version['version'] . '/t3x/',
 				'type' => 't3x',
 			),
+			'autoload' => array(
+				'classMap' => array('')
```

Comment:
Are you sure it's not `classmap`? See [schema](https://getcomposer.org/doc/04-schema.md#classmap).